app/controllers/movies_controller.rb

```diff
@@ -31,6 +31,15 @@ def new
   def create
     @movie = Movie.create(movie_params)
     @movie.user_id = current_user.id
+
+    unless ENV['RAILS_ENV'] == 'test'
+      externalMovie = search_for_movie(params[:movie][:name])
+      if externalMovie
+        @movie.imdbID = externalMovie["imdbID"]
+        @movie.remote_poster_url = poster_url(@movie.imdbID)
+      end
+    end
+
```

Comment:
Im going to allow this, but then STRONGLY suggest you look into mocking this API call yourself